nuget.config

```diff
@@ -4,6 +4,7 @@
     <clear />
```

Comment:
@TravisEz13, test results were not published for PowerShell-CI-macos at vstfs:///Build/Build/26449